lib/types/orders.d.ts

```diff
@@ -222,6 +226,58 @@ export declare namespace Orders {
             refund_speed: 'optimum' | 'normal';
         }
     }
+
+    interface ConvenienceFeeConfig {
```

Comment:
Why is this getting included in this PR? Does this have any relation with changing content-type to `application-json` @ankitdas13

Reply:
Reverted